forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -0,0 +1,190 @@
+package com.joinforage.forage.android.pos
+
+import com.joinforage.forage.android.CapturePaymentParams
+import com.joinforage.forage.android.CheckBalanceParams
+import com.joinforage.forage.android.DeferPaymentCaptureParams
+import com.joinforage.forage.android.ForageConfigNotSetException
+import com.joinforage.forage.android.ForageSDK
+import com.joinforage.forage.android.ForageSDKInterface
+import com.joinforage.forage.android.TokenizeEBTCardParams
+import com.joinforage.forage.android.core.telemetry.Log
+import com.joinforage.forage.android.network.model.ForageApiResponse
+import com.joinforage.forage.android.ui.ForagePANEditText
+import com.joinforage.forage.android.ui.ForagePINEditText
+
+data class RefundPosPaymentParams(
+    val foragePinEditText: ForagePINEditText,
+    val paymentRef: String,
+    val amount: Float,
+    val reason: String,
+    val metadata: Map<String, String>? = null
+)
+
+/**
+ * Instantiating a ForageTerminalSDK starts a connection with Forage’s servers.
+ * @param posTerminalId The ID of the POS terminal that is using the Forage SDK.
+ */
+class ForageTerminalSDK(
+    private val posTerminalId: String
+) : ForageSDKInterface {
+    private var forageSdk: ForageSDKInterface = ForageSDK()
```

Comment:
nit: thoughts on not having a private variable called `forageSDK` and instead just doing `ForageSDK().someMethod` inline? Technically the `var` adds extra mutable state and is only saving two characters the `()`.

This is a nit pick though so feel free to disregard

Reply:
The reason I did this was to allow testing via dependency injection of a `MockForageSDK` as [seen here](https://github.com/teamforage/forage-android-sdk/pull/149/files#diff-d0d4f59823e995c582f1c16d6a704a761a6edd995b27d521eaf8b6baedbfc73dR34-R41)

If you have any cleaner ways around this, let me know!